keyboards/mb44/readme.md

```diff
@@ -0,0 +1,15 @@
+# MB-44
+
+![MB-44](https://imgur.com/sECiIHR.jpg)
+
+A short description of the keyboard/project
+
+* Keyboard Maintainer: [melonbred](https://github.com/melonbred)
+* Hardware Supported: The PCBs, controllers supported
+* Hardware Availability: Links to where you can find this hardware
+
+Make example for this keyboard (after setting up your build environment):
+
+    make mb44:default
+
```

Comment:
Consider adding instructions on resetting the board here.

Reply:
Instructions on resetting the board have been added to readme.md.